www/front_src/src/App.js

```diff
@@ -28,11 +32,35 @@ class App extends Component {
     this.setState({ isFullscreenEnabled: true });
   }
 
+  UNSAFE_componentWillMount = () => {
+    axios("internal.php?object=centreon_acl_webservice&action=getCurrentAcl")
```

Comment:
where is this webservice ?
should be "currentAcl" instead of "getCurrentAcl"

Reply:
I have this webservice. @victorvassilev send me that one yesterday and it works :)

Reply:
ok so commit it in this branch :)
cause if we merge your PR, the application will be broken